.github/workflows/build-apk.yml

```
@@ -6,6 +6,10 @@ on:
       version-name:
         required: true
         type: string
+      build-type:
+        required: true
+        type: string
+        default: "release"
```

actionlint (1.7.7)

input "build-type" of workflow_call event has the default value "release", but it is also required. if an input is marked as required, its default value will never be used

(events)